src/golem-network.ts

```diff
@@ -100,6 +104,8 @@ export class GolemNetwork {
         basePath: this.options.api?.url,
       });
 
+      this.gftpStorageProvider = new GftpStorageProvider();
```

Comment:
this should be dependent on `dataTransferProtocol` option

Reply:
I've applied this remark. The only issue is that this capability is not really implemented in the WS-based GFTP transport:

```
Failed to run example on Golem GolemInternalError: Not implemented
    at WebSocketBrowserStorageProvider.publishFile (/home/ggodlewski/proj/golem/golem-js/dist/shared-ChCc8mz0.mjs:719:15)
    at GftpServerAdapter.publishFile (/home/ggodlewski/proj/golem/golem-js/dist/shared-ChCc8mz0.mjs:5963:44)
    at MarketModuleImpl.applyLocalGVMIServeSupport (/home/ggodlewski/proj/golem/golem-js/dist/shared-ChCc8mz0.mjs:3528:139)
    at MarketModuleImpl.buildDemand (/home/ggodlewski/proj/golem/golem-js/dist/shared-ChCc8mz0.mjs:3501:39)
    at <anonymous> (/home/ggodlewski/proj/golem/golem-js/examples/local-image/serveLocalGvmi.ts:48:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  previous: undefined
}
```